internal/subscription/core.go

```diff
@@ -112,6 +113,8 @@ func (c *Core) CreateSubscription(ctx context.Context, m *Model) error {
 		return err
 	}
 
+	fmt.Println(topicModel)
```

Comment:
Why are we using fmt.Println() and not logger.Ctx(ctx).Infow()

Reply:
Removing this. Added for local testing.